src/classification/main.jl

```diff
@@ -329,10 +336,10 @@ function apply_tree_proba(
         return apply_tree_proba(tree.right, features, labels)
     end
 end
-apply_tree_proba(tree::Root{S, T}, features::AbstractMatrix{S}, labels) where {S, T} =
-    apply_tree_proba(tree.node, features, labels)
-apply_tree_proba(tree::LeafOrNode{S, T}, features::AbstractMatrix{S}, labels) where {S, T} =
-    stack_function_results(row->apply_tree_proba(tree, row, labels), features)
+apply_tree_proba(tree::Root{S, T}, features::AbstractMatrix{S}, labels; use_multithreading = false) where {S, T} =
+    apply_tree_proba(tree.node, features, labels, use_multithreading = use_multithreading)
+apply_tree_proba(tree::LeafOrNode{S, T}, features::AbstractMatrix{S}, labels; use_multithreading = false) where {S, T} =
+    stack_function_results(row->apply_tree_proba(tree, row, labels), features, use_multithreading = use_multithreading)
```

Comment:
```suggestion
    stack_function_results(row->apply_tree_proba(tree, row, labels), features; use_multithreading)
```
Lower bound is set to 1.6 so no need to repeat the keyword name